### PR TITLE
Update Button.cpp

### PR DESCRIPTION
**Pull request by c0zm0z**
sloeber eclipse arduino esp32 will stop compiling, if there is no return at the end of this function.

### CODE REVIEW

**Comment by c0zm0z** [comment]
add "return -1;" in Button::update()
for compiler error message.

File: src/Button/Button.cpp [hunk 1]
```diff
@@ -88,7 +88,8 @@ boolean Button::update(){
 	} else if(_new_button_state){
 		_button_held();
 		return false;	// State hasn't changed
```
Comment (r89m) [inline src/Button/Button.cpp:90]: Could you move this line to where you put return -1 and I'll merge

**Review by r89m — CHANGES_REQUESTED** [review]

**Comment by r89m** [comment]
Thanks for the fix. If you move the line mentioned in the comment I'll merge it in.

Richard

File: src/Button/Button.cpp (hunk 1 above)
Comment (c0zm0z) [reply src/Button/Button.cpp]: done

Comment (c0zm0z) [reply src/Button/Button.cpp:90]: did already. Can you see my first coment. There was a updated Button.cpp

**Pull request merged**